src/testcases/org/apache/poi/ss/usermodel/BaseTestRow.java

```diff
@@ -466,4 +466,71 @@ public void testRowStyle() throws IOException {
        
        wb2.close();
     }
+    
+    @Test
+    public void testCellShiftingRight(){
+        Workbook wb = _testDataProvider.createWorkbook();
+        Sheet sheet = wb.createSheet("sheet1");
+        Row row = sheet.createRow(0);
+        row.createCell(0, CellType.NUMERIC).setCellValue(0);
+        row.createCell(1, CellType.NUMERIC).setCellValue(1);
+        row.createCell(2, CellType.NUMERIC).setCellValue(2);//C
+        row.createCell(3, CellType.NUMERIC).setCellValue(3);//D
+        row.createCell(4, CellType.NUMERIC).setCellValue(4);//E
+        row.createCell(5, CellType.NUMERIC).setCellValue(5);//F
+        row.createCell(6, CellType.NUMERIC).setCellValue(6);//G
+        try{
+            row.shiftCellsLeft(6, 4, 2); // range [6-4] is illegal
```

Comment:
shouldn't this test be using shiftCellsRight?

Reply:
this test fails for TestSXSSFCell (likewise for shiftLeft test) - they are inherited from this base class and SXSSF throws NotImplemented exceptions for this functionality

Reply:
If you could override the tests in TestSXSSFRow to stop the shift cell tests from running

Reply:
Just pushed that (https://github.com/zmau/poi/commit/40a0f1b595cf2a0bdeb26d56194c0b9bac31dbde).